examples_tests/02.ComputeShader/main.cpp

```diff
@@ -594,7 +586,9 @@ int main()
 	video::ISwapchain* rawPointerToSwapchain = swapchain.get();
 	
 	uint32_t currentFrameIndex = 0u;
-	while (!windowShouldClose_Global)
+	// while (!windowShouldClose_Global)
+	// for (uint32_t i = 0u; i < 1000u; ++i)
+	do
```

Comment:
this disgusts me, isn't there a function telling if we received a window close event in the window callback?

Reply:
yes, I think there is

this will get removed shortly

Reply:
resolved